### PR TITLE
Remove PGP encryption

### PR DESCRIPTION
**Pull request by saliceti**
## What

Instead of encrypting with GPG and sending the tokens via email we now
will use of a unique download link. We don't need GPG anymore so we
remove code, tests, model, etc.

## How to review

* Reset the DB:`rails db:reload`
* Run tests: `rspec`

### CODE REVIEW

File: app/controllers/access_requests_controller.rb [hunk 1]
```diff
@@ -13,7 +13,7 @@ def create
     @access_request = AccessRequest.new(access_request_params)
 
     if @access_request.save
-      Notify.service_team(@access_request, admin_access_request_url(@access_request))
```
Comment (himal-mandalia) [inline app/controllers/access_requests_controller.rb]: Accidentally left commented out methinks.

File: app/services/notify.rb [hunk 2]
```diff
@@ -8,6 +8,7 @@ module Notify
   TEAM_EMAIL = ENV['TEAM_EMAIL']
 
   def service_team(access_request, link)
+    byebug
```
Comment (himal-mandalia) [inline app/services/notify.rb]: Bye bye byebug?

File: app/views/access_requests/_form.html.haml [hunk 3]
```diff
@@ -29,16 +29,3 @@
...
-      = f.label :pgp_key_file, class: 'form-label' do
-        Your PGP/GPG public key
-        %span.form-hint
-          %p
-            Attach your GPG public key. #{link_to 'Guide to generating a GPG key', 'https://www.gnupg.org/gph/en/manual.html#AEN26', target: '_blank'}.
-
-          %p
-            This will be used to encrypt your token so we can send it to you securely.
-      = f.file_field :pgp_key_file
-  .actions
```
Comment (himal-mandalia) [inline app/views/access_requests/_form.html.haml]: We've lost the form actions / submit buttons.

File: app/views/admin/tokens/_form.html.haml [hunk 4]
```diff
@@ -32,21 +32,3 @@
...
-        = f.label :pgp_key_file, class: 'form-label' do
-          PGP/GPG public key
-          %span.form-hint
-            %p
-              This will be used to encrypt the token so it can be sent securely.
-        = f.file_field :pgp_key_file
-        = f.hidden_field :pgp_key, value: @token.pgp_key
-
-  .actions
-    = f.submit @token.new_record? ? 'Create' : 'Save', class: 'button'
```
Comment (himal-mandalia) [inline app/views/admin/tokens/_form.html.haml]: Actions / submit button lost.

**Review by himal-mandalia — CHANGES_REQUESTED** [review]

**Comment by saliceti** [comment]
Fixed

**Review by himal-mandalia — APPROVED** [review]

**Pull request merged**